docs/how-to-use-flatbuffers.md

```diff
@@ -1,8 +1,8 @@
 # How to use FlatBuffers
 
-Flatbuffers is used to serialise and deserialize some data structures.
+Flatbuffers is used to serialize and deserialize some data structures.
```

Comment:
I feel like I should object to this as the previous spelling was correct 😄

Reply:
well, if the word following was spelled the same, I wouldn't have changed it